S/SCALE_MAMBA/build_tarballs.jl

```diff
@@ -41,10 +45,7 @@ cp Setup.x${exe} ${prefix}/bin/
 """
 
 # Only x86_64, no FreeBSD or windows, and no musl
-platforms = [p for p in supported_platforms() if arch(p) == "x86_64" && !Sys.isfreebsd(p) && !Sys.iswindows(p) && libc(p) != "musl"]
-
-# Build with GCC 6 at least, to dodge C++ problems
-platforms = BinaryBuilder.replace_gcc_version.(platforms, :gcc6)
+platforms = supported_platforms(; exclude=p -> arch(p) != "x86_64" || Sys.isfreebsd(p) || Sys.iswindows(p) || libc(p) == "musl")
```

Comment:
When you exclude 14 out of 16 platforms, isn't
```suggestion
platforms = [Platform("x86_64", "linux"), Platform("x86_64", "macos")]
```
much simpler?